grommet/grommet
# add new fill prop to checkbox

#### What does this PR do?
Orginal work was done [here](https://github.com/grommet/grommet/pull/5019) 

This PR adds a new `fill` prop to `checkbox` 
When a user includes the `fill` prop the box containing the label and checkbox/toggle will fill the width and set `justify-content:  space-between`
#### Where should the reviewer start?
styledcheckbox.js
#### What testing has been done on this PR?
storybook and added a test for using the fill prop
#### How should this be manually tested?
```

        <FormField
          label="Toggle fill"
          name="toggle"
          htmlFor="check-box-toggle"
          {...props}
        >
          <Box pad={{ horizontal: 'small', vertical: 'xsmall' }}>
            <CheckBox
              id="check-box-toggle"
              name="toggle"
              label="CheckBox"
              toggle
              fill
              reverse
            />
          </Box>
        </FormField>
```
#### Any background context you want to provide?
feature request: see below
<img width="506" alt="Screen Shot 2021-04-08 at 9 42 29 AM" src="https://user-images.githubusercontent.com/42451602/114056285-be268a80-984e-11eb-9515-ee97ff2b046f.png">

#### What are the relevant issues?
closes  #5001 
#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
already done
#### Should this PR be mentioned in the release notes?
yes
#### Is this change backwards compatible or is it a breaking change?
backwards compatible
